src/solvers/dgsem_p4est/dg_2d_parallel.jl

```diff
@@ -18,6 +18,10 @@ function rhs!(du, u, t,
   @trixi_timeit timer() "prolong2mpiinterfaces" prolong2mpiinterfaces!(
```

Comment:
I haven't checked this file in detail since it seems to be mostly copy-paste from the 2D ParallelTreeMesh. If there's a function I should check in particularly, please let me know.

Reply:
Indeed, it's mostly copy-paste, so I think it's not necessary to check every little detail.